README.md

```diff
@@ -25,39 +25,45 @@ jsdoc -c conf.json -R README.md
 You can set options for customizing your documentations. Notice the `"template"` field for setting the path to **pixi-jsdoc-template**.
 
 ```json
-"templates": {
-    "applicationName": "Demo",
-    "disqus": "",
-    "googleAnalytics": "",
-    "favicon": "path/to/favicon.png",
-    "openGraph": {
-        "title": "",
-        "type": "website",
-        "image": "",
-        "site_name": "",
-        "url": ""
-    },
-    "meta": {
-        "title": "",
-        "description": "",
-        "keyword": ""
-    },
-    "linenums": true,
-    "source": {
-        "include": [
-            "./src/"
-        ],
-        "includePattern": ".+\\.js(doc)?$",
-        "excludePattern": "(^|\\/|\\\\)_"
-    },
-    "opts": {
-        "encoding": "utf8",
-        "recurse": true,
-        "private": false,
-        "lenient": true,
-        "destination": "./docs",
-        "template": "./node_modules/@pixi/jsdoc-template"
-    }
+{
+  "templates": {
+      "applicationName": "Demo",
+      "disqus": "",
+      "googleAnalytics": "",
+      "favicon": "path/to/favicon.png",
+      "openGraph": {
+          "title": "",
+          "type": "website",
+          "image": "",
+          "site_name": "",
+          "url": ""
+      },
+      "meta": {
+          "title": "",
+          "description": "",
+          "keyword": ""
+      },
+      "linenums": true,
+      "source": {
+          "include": [
+              "./src/"
+          ],
+          "includePattern": ".+\\.js(doc)?$",
+          "excludePattern": "(^|\\/|\\\\)_"
+      },
+      "opts": {
+          "encoding": "utf8",
+          "recurse": true,
+          "private": false,
+          "lenient": true,
+          "destination": "./docs",
+          "template": "./node_modules/@pixi/jsdoc-template"
+      }
+  },
+  "pixi": {
+      "styles": ["path/to/custom/styles.css"], // Optionally include custom styling
```

Comment:
Could you remove the comments (they are pretty self explanatory) or change the `json` to `js` as the language above